src/sbvr-api/uri-parser.ts

```diff
@@ -292,12 +308,14 @@ export async function parseOData(
 			const odata = memoizedParseOdata(url);
 
 			return {
+				id: b.id,
+				headers: { ...headers, ...b.headers } ?? headers,
```

Comment:
Can the `?? headers` ever take effect?
```suggestion
				headers: { ...headers, ...b.headers },
```